src/applications/representative-search/components/results/ReportModal.jsx

```diff
@@ -82,8 +81,8 @@ const ReportModal = ({
     Object.keys(reportObject).forEach(prop => {
       if (reportObject[prop] !== null) {
         if (prop === 'phone') {
-          formattedReportObject.reports[snakeCase('phoneNumber')] =
-            reportObject.phone;
+          /* eslint-disable-next-line camelcase */
```

Comment:
The backend validation is expecting snake case for both keys and values, so it needs to be converted somewhere on the frontend. Disabling eslint (rather than using lodash) is more explicit about that intention.